dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNFictTwoWTransformerInterfaceIIDM.cpp

```diff
@@ -0,0 +1,319 @@
+//
+// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+/**
+ * @file  DYNTwoWTransformerInterfaceIIDM.cpp
+ *
+ * @brief Two windings transformer data interface : implementation file for IIDM implementation
+ *
+ */
+
+#include "DYNFictTwoWTransformerInterfaceIIDM.h"
+
+#include <powsybl/iidm/ThreeWindingsTransformer.hpp>
+
+#include "DYNCommon.h"
+#include "DYNPhaseTapChangerInterfaceIIDM.h"
+#include "DYNRatioTapChangerInterfaceIIDM.h"
+#include "DYNStepInterfaceIIDM.h"
+#include "DYNBusInterface.h"
+#include "DYNVoltageLevelInterface.h"
+#include "DYNStateVariable.h"
+#include "DYNModelConstants.h"
+
+#include "DYNTrace.h"
+
+using boost::shared_ptr;
+using std::string;
+using std::vector;
+
+namespace DYN {
+
+  FictTwoWTransformerInterfaceIIDM::FictTwoWTransformerInterfaceIIDM(const std::string& Id,
+                      stdcxx::Reference<powsybl::iidm::ThreeWindingsTransformer::Leg>& leg,
+                      const bool& initialConnected1, const double& VNom1, const double& ratedU1) :
```

Comment:
no need to have const ..& on bool and double

Reply:
ok, done